### DISCUSSION TITLE
2008/07/14/%E5%9C%A8-google-lively-%E4%B8%8A%E5%BB%BA%E4%BA%86%E4%B8%80%E4%B8%AA%E6%88%BF%E9%97%B4

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#231] [Announcements]
# 2008/07/14/%E5%9C%A8-google-lively-%E4%B8%8A%E5%BB%BA%E4%BA%86%E4%B8%80%E4%B8%AA%E6%88%BF%E9%97%B4

欢迎大伙来我的房间做客啊:  (点击下面我房间的图片)

https://blog.qizhen.xyz/2008/07/14/%E5%9C%A8-google-lively-%E4%B8%8A%E5%BB%BA%E4%BA%86%E4%B8%80%E4%B8%AA%E6%88%BF%E9%97%B4

**Comment by ruanqizhen** [comment]
Gene
2008年07月14日 09:45 编辑
很可愛的說^^

回复
